### ISSUE TITLE
Add hotkey for loading waypoints

### ISSUE THREAD

**Issue by Gil80** [rkusa/dcs-scratchpad#71]
Is it possible to add a new hotkey for loading waypoints to the F18?
A hotkey that triggers the Load Waypoints / Load Coordinates, instead of using the mouse to click the button.

**Comment by rkusa** [comment]
Yes, there is an extension for that, please check the following conversation for details: https://github.com/rkusa/dcs-scratchpad/issues/69#issuecomment-2442016807

**Comment by Gil80** [comment]
Yes, that was my question from October. I wasn't able to make it work.
Based on the response I got from @aeronautes in this link: https://github.com/aeronautes/dcs-scratchpad/discussions/1#discussioncomment-11676140

> A hotkey for LatLon will require a change to scratchpad-hook.lua as the window table is not shared with extensions and there is no scratchpad API to request hotkey assignments from extensions. So it's partly getting a PR put together for rkusa to look at.

**Comment by rkusa** [comment]
Oops, sorry didn't look at the username careful enough. Thank you for the clarification, I'll reopen the issue. Though I am not entirely sure if it is worth adding a shortcut for something like that. Would it be that much better than simply pressing a button?

**Comment by aeronautes** [comment]
I would say the functionality attached to a hotkey can vary widely as in the case of the apit extension, which can be used as a programmable macro interfacing to cockpit systems. In this particular example it's a matter of convenience perhaps but in other contexts flying nap of earth with both hands on controls, manipulating a mouse and having scratchpad on screen may be prohibitive.

**Comment by Gil80** [comment]
> Oops, sorry didn't look at the username careful enough. Thank you for the clarification, I'll reopen the issue. Though I am not entirely sure if it is worth adding a shortcut for something like that. Would it be that much better than simply pressing a button?

That's a fair question.
Given that I'm a VR user, I try to bind most of the keys to my HOTAS and my Razer Tartarus V2 which sits on my lap.
I think that after you're getting used to ctrl+shift+x to open and then be able to bind something like ctrl+shift+i to insert, then one would hope for ctrl+shift+[whatever] to add them to the cockpit.

**Comment by Gil80** [comment]
@aeronautes can you please have a look at the attached code?
I think you can merge some of the things from this. One of the biggest changes is the tick rate. This version of scratchpad enters the coordinates in split seconds. I honestly prefer your version, as it has more to it, but the speed in this version with the tick rate settings is a real treat.
Additionally, this version enters the coordinates no matter how your DDI is set up, as long as you have precise mode enabled.
I hope you can have a look and cherry pick the good stuff and merge with your own :)

[dcs-scratchpad-main.zip](https://github.com/user-attachments/files/18292531/dcs-scratchpad-main.zip)
